Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM rust:1.67 as builder
+
+RUN apt update
```

Comment:
This isn't needed as you are using the cargo package manager

Reply:
Might not be worth the effort as it only saves <1GB

eard-debian       latest    5d7446f83b66   21 seconds ago   3.84GB
eard-alpine        latest    caaf1a478f55      8 minutes ago     3.09GB